internal/app/push/cli.go

```diff
@@ -273,7 +273,9 @@ func (c idToPushConverter) getPlan(id id.IngressDescriptor) push.Plan {
 
 	for idx := uint(0); idx < id.Relations().Len(); idx++ {
 		rel := id.Relations().Relation(idx)
-		relations = append(relations, c.getRelation(rel.Name()))
+		if rel.LookUpChild() || rel.LookUpParent() {
```

Comment:
We should check the full path from the start table.

For example if one relation is deactivated (BC) all relations after should be deactivated (CD).

```
A --> B -\-> C --> D
```